plugin.go

```diff
@@ -211,3 +215,17 @@ func remapSubmodule(name, url string) *exec.Cmd {
 		url,
 	)
 }
+
+// setProtocol returns a git command that, when executed, configures git to
+// override the default protocol.
+func setProtocol(source string, dest string) *exec.Cmd {
+	destination := "url.\"" + dest + "\".insteadOf"
```

Comment:
This is not working for me, in the docker image seems to be using a git version where the quotes does not work and I'm getting this error:
`fatal: I don't handle protocol '"https'`
Changing this line to this

```go
	destination := "url." + dest + ".insteadOf"
```
Seems to work correctly